.vortex/CLAUDE.md

```
@@ -291,18 +291,23 @@
 yarn test:watch     # Watch mode for development
 ```
 
-**Template Testing** (`.vortex/tests/`):
+**Template Testing** (`.vortex/`):
 ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)